internal/controller/exporter_controller.go

```diff
@@ -56,79 +54,103 @@ type ExporterReconciler struct {
 func (r *ExporterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
 
-	exporter := &jumpstarterdevv1alpha1.Exporter{}
-	err := r.Get(ctx, req.NamespacedName, exporter)
-	if apierrors.IsNotFound(err) {
-		logger.Info("reconcile: Exporter deleted", "exporter", req.NamespacedName)
-		// Request object not found, could have been deleted after reconcile request.
-		// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
-		return reconcile.Result{}, nil
+	var exporter jumpstarterdevv1alpha1.Exporter
+	if err := r.Get(ctx, req.NamespacedName, &exporter); err != nil {
+		logger.Error(err, "Reconcile: unable to get exporter", "exporter", req.NamespacedName)
```

Comment:
This is going to log an error when a exporter is deleted, we are bumping severity from info to Error